Spring boot Security Basics

dmytro-voronin opened the pull request.

dmytro-voronin committed:
Add Security

Sanderfuck commented on src/main/java/mate/academy/bookstore/dto/user/UserRegistrationRequest.java:
```diff
@@ -0,0 +1,28 @@
+    private String password;
+    private String repeatPassword;
+    @NotBlank
+    @Size(min = 1, max = 30)
+    private String firstName;
+    @NotBlank
+    @Size(min = 1, max = 40)
+    private String lastName;
```
One letter of firstname and lastname not looks like in real life

Sanderfuck commented on src/main/java/mate/academy/bookstore/exception/CustomGlobalExceptionHandler.java:
```diff
@@ -18,6 +18,18 @@
+            WebRequest request
+    ) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", LocalDateTime.now());
+        body.put("status", HttpStatus.BAD_REQUEST);
+        body.put("message", ex.getMessage());
```
You can save this strings in constants

dmytro-voronin committed:
change @Size firstName min = 3
change @Size lastName min = 3
in UserRegistrationRequest
added static String TIMESTAMP, STATUS, MESSAGE

kozhukhovsky commented on src/main/java/mate/academy/bookstore/dto/user/UserRegistrationRequest.java:
```diff
@@ -0,0 +1,28 @@
+    @NotBlank
+    @Size(min = 5, max = 200)
+    private String shippingAddress;
+
```
```suggestion
```
